src/xml.tsx

```diff
@@ -272,6 +272,13 @@ export function parse(source: string, middleware?: Middleware): JsxAST | null {
     );
   }
 
+  function decodeEntities(text: string): string {
+    return text
+      .replace(/&amp;/g, '&')
+      .replace(/&lt;/g, '<')
+      .replace(/&gt;/g, '>');
```

Comment:
Making an exception for some character just seems wrong. What about `&apos;` or `&quot;`? There are dozens of characters that could be escaped.